lib/src/core/metadata/directives.dart

```diff
@@ -722,7 +727,7 @@ class DirectiveMetadata extends InjectableMetadata {
 ///
 /// ## Lifecycle hooks
 ///
-/// When the component class implements some [../../guide/lifecycle-hooks.html]
+/// When the component class implements some [lifecycle-hooks](docs/guide/lifecycle-hooks.html)
```

Comment:
We surely do. Can you make a proper page for that, with copy and stuff? I can put it to dartdocs.org.